### PR TITLE
Delegate hotfix version 81015 for CDS-82385, ZD-52612

### PR DESCRIPTION
**Pull request by GoelPratyush99**
Added hotfix description for 81015. The increment in build number `81010` to `81015` involved 2 changes corresponding to these tickets https://harness.atlassian.net/browse/CDS-82385 and https://harness.atlassian.net/browse/CI-9707. Hence added release notes for both of them

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/delegate.md [hunk 1]
```diff
@@ -119,7 +119,13 @@ This release does not include any new early access features.
 
 ##### Hotfixes
 
-This release does not include hotfixes.
+#### Version 81015
+
+- Fixed an issue that prevented the service dashboard from showing the new active instance count resulting from updates made to workload replicas. The issue occurred in a few Helm deployment scenarios, when the updates were made after deployment. (CDS-76795, ZD-52612)
```
Comment (deepakbolangady) [inline release-notes/delegate.md]: Hi, Pratyush. The descriptions needed rewording (to fix some typos and clarity). I met with Soumyajit and enhanced the description for the CI ticket (CI-9707). When taking a look at this description for CDS-76795, I noticed that the ZenDesk IDs on the Jira ticket and the one in the PR are not the same. Also, CDS-76795 already has a description in Jira that does not match the one in the PR. Please check and let me know how to proceed. Thanks.

**Review by deepakbolangady — APPROVED** [review]
Approving after reworking the descriptions in collaboration with Soumyajit and Pratyush.

**Review by deepakbolangady — APPROVED** [review]

**Review by brian-f-harness — DISMISSED** [review]

**Review by brian-f-harness — DISMISSED** [review]

**Review by brian-f-harness — APPROVED** [review]

**Review by brian-f-harness — DISMISSED** [review]

**Review by pranaykaikini — APPROVED** [review]

**Comment by brian-f-harness** [comment]
@GoelPratyush99 Is this hotfix still in progress, or can we close this PR?

**Comment by GoelPratyush99** [comment]
@brian-f-harness We can publish for delegates. We have hot fix cg manager versions `81311`, `81405`and `81504` for the same issue.

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**